### PR TITLE
Adjustbox

### PR DESCRIPTION
**Pull request by brucemiller**
Binding for adjustbox.

Initially, just the groundwork that allows `adjustbox.sty` to be processed without errors:  a few random latex defintions; processing registers correctly w.r.t `\protect`; expansion of keyvals parameters and general KeyVal cleanup and simplification.

Next up is getting the correct (visual) results from adjustbox.

Of interest to @dginev, @tkw1536

### CODE REVIEW

File: lib/LaTeXML/Package/xkeyval.sty.ltxml [hunk 1]
```diff
@@ -20,16 +20,17 @@ use LaTeXML::Util::Pathname qw(pathname_split);
 ###
 ### Basic \setkeys
 ###
+AssignValue('keyval.sty_loaded' => 1, 'global');    # pretend keyval loaded too.
```
Comment (dginev) [inline lib/LaTeXML/Package/xkeyval.sty.ltxml:23]: is it loading in the pdflatex flow? If so, why not `RequirePackage` instead?

File: lib/LaTeXML/Package/graphics.sty.ltxml [hunk 2]
```diff
@@ -24,14 +24,17 @@ use LaTeXML::Util::Image;
 
 DefParameterType('GraphixDimension', sub {
     my ($gullet) = @_;
-    if ($gullet->ifNext(T_OTHER('!'))) {
-      $gullet->readToken();
+    $gullet->skipSpaces;
+    my $next = $gullet->readXToken;
+    # Wrong !?!?!
```
Comment (dginev) [inline lib/LaTeXML/Package/graphics.sty.ltxml:29]: curious if there are details :>

**Review by dginev — COMMENTED** [review]
Exciting!

Not much to comment for now on my side. The `readXToken` bits will be interesting to flesh out fully while keeping everything else happy.

File: lib/LaTeXML/Package/xkeyval.sty.ltxml (hunk 1 above)
Comment (brucemiller) [reply lib/LaTeXML/Package/xkeyval.sty.ltxml:23]: It's really just to keep `keyval` from being loaded later and overwriting `xkeyval` defs (the latter is proper superset)

File: lib/LaTeXML/Package/graphics.sty.ltxml (hunk 2 above)
Comment (brucemiller) [reply lib/LaTeXML/Package/graphics.sty.ltxml:29]: just wondering if interpretation is correct

**Comment by brucemiller** [comment]
ok, let's call that done;; modulo reviews, CI, etc

**Pull request merged**